operatorcourier/verify.py

```diff
@@ -0,0 +1,131 @@
+import os
+import logging
+from operatorcourier.build import BuildCmd
+from operatorcourier.validate import ValidateCmd
+from operatorcourier.errors import OpCourierBadBundle
+from operatorcourier import identify
+from typing import List, Dict
+import semver
+
+logger = logging.getLogger(__name__)
+
+
+def get_manifests_info(source_dir: str, yamls: List) -> Dict:
+    """
+    Given a source directory OR a list of yaml files. The function returns a dictionary
+    containing all operator bundle file information grouped by version. Note that only
+    one of source_dir or yamls can be specified.
+
+    :param source_dir: Path to local directory of operator bundles, which can be either
+                       flat or nested
+    :param yamls: A list of yaml strings to create bundle with
+    :return: A dictionary object where the key is the the semantic version of each bundle,
+             and the value is a list of yaml strings of operator bundle files.
+             If the directory is flat, the key of yaml strings will be an empty string.
+    """
+    def get_manifest_files_content(file_paths: List) -> List:
```

Comment:
Could you please extract `get_manifest_files_content` and `is_dir_name_semver` from this function to module? I don't see reason why it needs closure.